build/Dockerfile.gobmp

```diff
@@ -1,4 +1,6 @@
-FROM scratch
+FROM alpine:3.14
```

Comment:
It is a bad sign that we need to customise the image to support healthchecks.

Here is a nice post, about avoiding the use of `curl` for healthchecks: https://blog.sixeyed.com/docker-healthchecks-why-not-to-use-curl-or-iwr/
This is food-for-thought. In general, I believe we should avoid mixing `microservice` responsibility with health checks that inherently belong to the orchestrator.

Reply:
yes I had considered that, but what would be a nice alterative? Both `gobmp` and `pg2kafka` simply spawn a main go routine. So the least intrusive part is to spawn a parallel HTTP server from this routine which we can query directly. I opted for `curl` to avoid all the hurdle of implementing a custom healthcheck and change much of the in-app logic. Any recommendation on this is more than welcome. If needed, we can start with HTTP and migrate to sth more app-specific separately.